LICENSE-CODE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 GitHub
+Copyright ©️ 2022 GitHub
```

Comment:
```suggestion
Copyright 2022 GitHub
```

According to https://opensource.org/licenses/MIT it should be:
```
Copyright <YEAR> <COPYRIGHT HOLDER>
```
no need to use `(c)` or `©️` because it's already there on the first word.